manifests/plugin/abrt/params.pp

```diff
@@ -1,6 +1,7 @@
 # Default parameters for the Pulp smart proxy plugin
 class foreman_proxy::plugin::abrt::params {
   $enabled                 = true
+  $version                 = 'installed'
```

Comment:
I'm a little lost with this. What's the point in keeping abrt::params::version, since it will never be used? Should plugin::abrt::version default to foreman_proxy::plugin::version?